src/recipe/test/unit/recipe.service.spec.ts

```diff
@@ -160,7 +160,7 @@ describe('Recipe Service', () => {
       limit: 2,
     });
 
-    expect(queryBusExecuteFindMany).toBeCalledTimes(1);
+    expect(queryBusExecuteFindMany).toBeCalledTimes(2);
```

Comment:
name of that spy indicates that you spy on a particular query (queryBusExecuteFindMany) from queryBus, in fact, you spay on 'execute'. Please consider renaming this const